### PR TITLE
feature(editors): remove note-, code- and base-editor including their node dependencies

### PR DESCRIPTION
**Pull request by nappynapster**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser-Support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser-Support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: src/lib/components/forms/dynamic-form/dynamic-switch/terra-dynamic-switch.component.html [hunk 1]
```diff
@@ -145,22 +145,6 @@
...
-                </terra-note-editor>
-                <terra-code-editor
-                    *ngSwitchCase="_controlTypeEnum.CODE_EDITOR"
-                    [formControlName]="formField.key"
-                    [inputFixedHeight]="formField.fixedHeight"
-                    [inputHeaderLabel]="formField.label"
-                    [(ngModel)]="formField.defaultValue"
-                >
-                </terra-code-editor>
-
```
Comment (pweyrich) [inline src/lib/components/forms/dynamic-form/dynamic-switch/terra-dynamic-switch.component.html:163]: Oh je.. Die Dynamic-Form haben wir ja auch noch.. Die muss dann bei dem Major Release auch rausfliegen..

**Review by pweyrich — COMMENTED** [review]
Einen Changelog müsstest du noch schreiben.
Und die dependencies auch noch aus der _lib/package.json_ entfernen.

**Comment by pweyrich** [comment]
`@types/htmlhint` kannst du übrigens auch noch deinstallieren

**Review by pweyrich — DISMISSED** [review]

**Review by pweyrich — DISMISSED** [review]
NOTE: This PR must not be merged before the dynamic-form has been removed

**Review by niwolf — DISMISSED** [review]

**Comment by niwolf** [comment]
@nappynapster closed since, we have all in terra now :D

**Pull request closed without merging**